test/vagrant_cleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+vagrant destroy -f
+
+i=1
+while [ "$i" -le "$K8S_NODES" ]; do
```

Comment:
We don't seem to set `$K8S_NODES` in `jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile` (only in `jenkinsfiles/ginkgo-kernel.Jenkinsfile`) but the script is used in the runtime Jenkinsfile too. Do we need to set `$K8S_NODES` explicitly or am I missing something?

Reply:
:grimacing: I tested this a bit too quickly :see_no_evil: 

I'll go back to draft and retest after fixing.

Reply:
It's fixed now.